codice/alliance
# CAL-284 Update imaging-app to use imaging-nitf 0.8

#### What does this PR do?
The imaging-nitf fluent-api bundle was recently split into 'fluent' and 'fluent-api'.  The imaging-nitf core bundle was recently split into 'core-api' and 'core'. The bulk of these changes remove unneeded embedded dependencies from pom files and to make code changes to reflect the imaging-nitf API changes.

Imaging-nitf core also added the ability to deal with coordinates in UTM and MGRS formats. This PR also modifies a line of code that was filtering out locations that weren't in DMS or decimal degrees format.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@glenhein 
@roelens8 
@vinamartin 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jaymcnallie
@kcwire
@jlcsmith 

#### How should this be tested?
1) build and start Alliance
   -- verify that the imaging-app starts up
2) ingest some NITFs that have locations recorded with various coordinate systems (UTMNORTH, UTMSOUTH, MGRS, etc.)
   -- verify that the product appears on the globe at the correct location, regardless of coordinate system.

#### Any background context you want to provide?
See Description 

#### What are the relevant tickets?
[CAL-284] (https://codice.atlassian.net/browse/CAL-284)

#### Checklist:
